Item.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Aeon.ShoppingCart
+{
+    public class CartItem
+    {
+        [Key]
+        public string ItemId { get; set; }
+
+        public int Name { get; set; }
```

Comment:
I'm quite sure that it's more appropriate for names to be a string. Also, the file is left over in the root directory of the project.